src/internal/converters/eml/eml.go

```diff
@@ -152,8 +156,23 @@ func FromJSON(ctx context.Context, body []byte) (string, error) {
 				return "", clues.WrapWC(ctx, err, "invalid content bytes")
 			}
 
+			name := ptr.Val(attachment.GetName())
+
+			contentID, err := attachment.GetBackingStore().Get("contentId")
+			if err != nil {
+				return "", clues.WrapWC(ctx, err, "getting content id for attachment")
+			}
+
+			if contentID != nil {
+				cids, _ := contentID.(*string)
+				if len(*cids) > 0 {
```

Comment:
We have a helper for this
```suggestion
				cIDs, err := str.AnyToString(contentID)
				if len(cids) > 0 {
```